pandas/core/indexes/category.py

```diff
@@ -293,6 +304,23 @@ def values(self):
         """ return the underlying data, which is a Categorical """
         return self._data
 
+    @property
+    def _values(self):
+        return self._data
```

Comment:
Is there a difference between `._data` and `.values` ? As below you use `.values` for itemsize and nbytes, but both seems to point to the underlying Categorical

Reply:
Probably just me being inconsistent.